### PR TITLE
Add FrostWire

### PR DESCRIPTION
**Pull request by ngilles**
Add FrostWire #531

### CODE REVIEW

File: 01-main/README.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ cog.out(pretty_list)
 | [<img src="../.github/github.png" align="top" width="20" />](https://flameshot.org/) | `flameshot` | <i>Flameshot is a free and open-source, cross-platform tool to take screenshots with many built-in features to save you time.</i> |
 | [<img src="../.github/launchpad.png" align="top" width="20" />](https://johnfactotum.github.io/foliate/) | `foliate` | <i>A simple and modern eBook viewer for Linux desktops.</i> |
 | [<img src="../.github/github.png" align="top" width="20" />](https://meetfranz.com/) | `franz` | <i>A messaging app for WhatsApp, Facebook Messenger, Slack, Telegram and many many more.</i> |
+| [<img src="../.github/github.png" align="top" width="20" />](https://www.frostwire.com/) | `frostwire` | <i>An easy to use Cloud Downloader, BitTorrent Client and Media Player. Search, Download, Play, Share</i> |
```
Comment (natanjunges) [inline 01-main/README.md:68]: This file is not meant to be changed by PRs.

**Review by natanjunges — CHANGES_REQUESTED** [review]

File: 01-main/packages/frostwire [hunk 2]
```diff
@@ -0,0 +1,10 @@
+DEFVER=1
+ARCHS_SUPPORTED="amd64"
```
Comment (natanjunges) [inline 01-main/packages/frostwire:2]: This is already the default, so not needed.

**Review by natanjunges — CHANGES_REQUESTED** [review]

File: 01-main/README.md (hunk 1 above)
Comment (ngilles) [reply 01-main/README.md]: `main/01-main/CONTRIBUTING.md` still mentions to run `cog` against the `README.md` :-/ (although that does require to install the new files manually locally first for it to show up). Guess it should be removed?

Comment (natanjunges) [reply 01-main/README.md]: Yes, #625 already does that, but it was not merge yet.

**Comment by ngilles** [comment]
Ok, looks like GH is lagging a bit here, my last commit was [2c67623](https://github.com/ngilles/deb-get/commit/2c676232e239b55aa840dbedf1c6730907c04e59) which this PR briefly showed, but now seems to be back on the non-rebased version?

File: 01-main/packages/frostwire (hunk 2 above)
Comment (ngilles) [reply 01-main/packages/frostwire]: hmm, looks like the DEFVER=1 is required

**Review by flexiondotorg — APPROVED** [review]

**Pull request merged**